mlkem/cbmc.h

```diff
@@ -126,4 +128,9 @@
   }
 // clang-format on
 
+// Wrapper around ARRAY_IN_BOUNDS operating on absolute values
+#define ARRAY_ABS_BOUND(arr, lb, ub, k)                                      \
```

Comment:
This should probably be used in many more places. Can you open an issue for that, please?

Reply:
https://github.com/pq-code-package/mlkem-native/issues/379